src/CarData.php

```diff
@@ -33,7 +33,6 @@ public static function getVehicleGearBoxType() : array
         return static::$vehicleGearBox;
     }
 
-    // phpcs:disable
```

Comment:
Why did you remove this?

Reply:
It's useless. I don't see any purpose for that code, it's just commented, so I remove it. Any reason why it should not be removed?